agr_literature_service/lit_processing/dump_json_data.py

```diff
@@ -0,0 +1,576 @@
+            row['url'] = "https://doi.org/doi:" + x.curie.replace("DOI:", "")
+        elif x.curie.startswith('SGD:'):
+            row['pages'] = [{"name": "reference",
+                             "url": "https://www.yeastgenome.org/reference/" + x.curie.replace("SGD:", "")}]
+            row['url'] = "https://www.yeastgenome.org/reference/" + x.curie.replace("SGD:", "")
+        elif x.curie.startswith('WB:'):
+            row['pages'] = [{"name": "reference",
+                             "url": "https://www.wormbase.org/db/get?name=" + x.curie.replace("WB:", "") + ";class=Paper"}]
+            row['url'] = "https://www.wormbase.org/db/get?name=" + x.curie.replace("WB:", "")
+        elif x.curie.startswith('MGI:'):
+            row['pages'] = [{"name": "reference",
+                             "url": "http://www.informatics.jax.org/reference/" + x.curie}]
+            row['url'] = "http://www.informatics.jax.org/"
+        elif x.curie.startswith('FB:'):
+            row['pages'] = [{"name": "reference",
+                             "url": "https://flybase.org/reports/" + x.curie.replace("FB:", "") + ".html"}]
+            row['url'] = "https://flybase.org/reports/" + x.curie.replace("FB:", "") + ".html"
+        elif x.curie.startswith('ZFIN:'):
+            row['pages'] = [{"name": "reference",
+                             "url": "https://zfin.org/" + x.curie.replace("ZFIN:", "")}]
+            row['url'] = "https://zfin.org/" + x.curie.replace("ZFIN:", "")
+        elif x.curie.startswith('RGD:'):
+            row['pages'] = [{"name": "reference",
+                             "url": "https://rgd.mcw.edu/rgdweb/report/reference/main.html?id=" + x.curie}]
+            row['url'] = "https://rgd.mcw.edu/rgdweb/elasticResults.html?term=" + x.curie
+        data.append(row)
+        reference_id_to_xrefs[x.reference_id] = data
+        if x.curie.startswith('PMID:'):
+            reference_id_to_pmid[x.reference_id] = x.curie.replace("PMID:", "")
+
+    return (reference_id_to_xrefs, reference_id_to_pmid)
+
+
+def get_author_data(db_session, mod_id):
+
+    orcid_to_cross_ref = {}
+    for x in db_session.query(CrossReferenceModel).filter(CrossReferenceModel.curie.like('ORCID:%')).all():
+        orcid_to_cross_ref[x.curie] = (x.pages, x.is_obsolete)
+
+    reference_id_to_authors = {}
+    for x in db_session.query(AuthorModel).join(ReferenceModel.author).outerjoin(ReferenceModel.mod_corpus_association).filter(ModCorpusAssociationModel.mod_id == mod_id).order_by(AuthorModel.reference_id, AuthorModel.order).all():
+        data = []
+        if x.reference_id in reference_id_to_authors:
+            data = reference_id_to_authors[x.reference_id]
+        orcid = x.orcid
+        if x.orcid and x.orcid in orcid_to_cross_ref:
+            (pages, is_obsolete) = orcid_to_cross_ref[x.orcid]
+            orcid = {"curie": x.orcid,
+                     "url": "https://orcid.org/" + x.orcid.replace("ORCID:", ""),
```

Comment:
same thing about resourceDescriptors.yaml here about orcid, and below in line 415, and anywhere there might be something about cross_references.

Reply:
removing url fixes this, thanks Shuai